tools/clang/test/CodeGenSPIRV/intrinsics.abs.hlsl

```diff
@@ -0,0 +1,79 @@
+// Run: %dxc -T vs_6_0 -E main
+
+// CHECK:      [[glsl:%\d+]] = OpExtInstImport "GLSL.std.450"
+
+void main() {
+  float result;
+  float2 result2;
+  float3 result3;
+  float4 result4;
+  float3x2 result3x2;
+  int3 iresult3;
+
+// CHECK:      [[a:%\d+]] = OpLoad %float %a
+// CHECK-NEXT: [[FAbs_a:%\d+]] = OpExtInst %float [[glsl]] FAbs [[a]]
+// CHECK-NEXT: OpStore %result [[FAbs_a]]
```

Comment:
I feel we are checking too much in our existing FileCheck tests. The purpose of these tests is to check ext inst, so no need to check `OpStore` actually. This will increase test robustness. I have the same problem in my previous tests. We should ignore stuff not related to the features to be tested for the future tests. Just wanted to raise the issue. :)